async-nats/src/jetstream/mod.rs

```diff
@@ -132,6 +132,7 @@ use crate::Client;
 pub mod account;
 pub mod consumer;
 pub mod context;
+pub mod errors;
```

Comment:
Think this module can be flattened into response (with into_future requests, we'll probably be having a request module too).

```suggestion
pub mod response;
```

Reply:
Flattened as agreed on chat.
Error accessible via `async_nats::jetstream::Error`